test/docker/dockers/rundeck/Dockerfile

```diff
@@ -70,6 +70,13 @@ RUN sudo dos2unix $HOME/api_test/api/*.sh
 RUN sudo dos2unix $HOME/api_test/src/*.sh
 RUN sudo dos2unix /tests/*.sh
 
+RUN mkdir -p $HOME/resources && \
+    sudo chown rundeck:root $HOME/resources && \
+    chmod 775 $HOME/resources && \
+    sudo mkdir -p /test && \
```

Comment:
the test folder is being created on line 64, is it necessary to rerun the command before tweaking the folder permissions?

Reply:
You;re so right! i didn't see it!